test/lib/ansible_test/_data/sanity/validate-modules/validate_modules/main.py

```diff
@@ -940,14 +940,22 @@ def _validate_docs(self):
                     msg='No DOCUMENTATION provided'
                 )
             else:
-                documentation_exists = True
```

Comment:
Looking at this more closely, I realized this shouldn't be moved. It's used to indicate that the `DOCUMENTATION` variable exists, not that it contains anything. The check for `doc` that was added below is all that is needed to avoid the reported traceback.

Reply:
It looks like restoring this was overlooked in the last commit.

Reply:
Do not remove `documentation_exists = True` in this PR.